### DISCUSSION TITLE
Sorting playlists by number of participants

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#16982] [Ideas] [answered]
As more and more people switch over to lazer, more playlists will be created. Currently, the number of participants in a playlist is a good indicator of "quality" and popularity for each playlist. An additional "Participants" sorting option on top of Open, Recently Ended, Participated, and Owned personally seems like a good option to quickly find and play a "good" playlist.

P.S. This could also be expanded on to just adding more sorting options in general, e.g. Number of Beatmaps, Total Duration, Extra Mods, Star Rating, Time until Closed, etc.

**Comment by peppy** [comment, marked as answer]
For sure. I think we need more ability for the user to be able to change order, but I also don't think the default sort should be play count (which is why we haven't done that yet). Maybe a combination of that and other factors.

**Reply by ikkue** [reply]
Maybe I worded it incorrectly, but I was only suggesting an additional sorting option and not a new default one. Thank you for the answer anyways!
